src/dra/5298C.c

```diff
@@ -1411,7 +1411,35 @@ void func_800F9DD0(u8* arg0, u8* arg1) {
 
 INCLUDE_ASM("dra/nonmatchings/5298C", func_800F9E18);
 
+#if defined(VERSION_HD)
```

Comment:
Since we aren't scraping decomp.me for the HD version, could you add your link if it doesn't get solved?